processor/src/main/java/org/mobilitydata/gtfsvalidator/processor/TableLoaderGenerator.java

```diff
@@ -272,13 +269,14 @@ private MethodSpec generateLoadMethod() {
             ArrayList.class)
         .addStatement("boolean hasUnparsableRows = false")
         .addStatement(
-            "final $T singleEntityValidators = validatorLoader.createSingleEntityValidators("
-                + "$T.class, validationContext)",
+            "final $T singleEntityValidators = validatorProvider.createSingleEntityValidators("
+                + "$T.class)",
             ParameterizedTypeName.get(
                 ClassName.get(List.class),
                 ParameterizedTypeName.get(
                     ClassName.get(SingleEntityValidator.class), gtfsEntityType)),
             gtfsEntityType);
+
```

Comment:
nit - blank line

Reply:
The code looks more structured with it. We start a "for" cycle after that line.